alerts/notifications.py

```diff
@@ -0,0 +1,36 @@
+from django.core.mail import send_mail
+from users.models import User
+from series.models import APIShow
+
+
+class NotificationBackend:
+    """Backend for generic notifications."""
+
+    def notify(self, user: User, tv_show: APIShow):
+        pass
```

Comment:
Better would be better to `raise NotImplementedError`, as this is pretty much an abstract method (`NotificationBackend` being an interface). :-)